pymatgen/util/provenance.py

```diff
@@ -256,7 +256,7 @@ def __init__(
         if not all(sys.getsizeof(h) < MAX_HNODE_SIZE for h in history):
             raise ValueError(f"One or more history nodes exceeds the maximum size limit of {MAX_HNODE_SIZE} bytes")
 
-        self.created_at = created_at or datetime.datetime.utcnow()
+        self.created_at = created_at or datetime.datetime.now(datetime.timezone.utc)
```

Comment:
Need to track https://github.com/materialsvirtuallab/monty/pull/688 until a new release is issued.